system/runners/BaseRunner.cfc

```diff
@@ -195,7 +195,7 @@ component {
 		required testResults,
 		required targetMD
 	){
-		var pathPatternMatcher = new testbox.system.util.PathPatternMatcher();
+		var pathPatternMatcher = new testbox.system.modules.globber.models.PathPatternMatcher();
```

Comment:
Is it ok to reach into a module sub-folder like this?  [NPM's package.json has a "main" field](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files), but I don't see anything like that in box.json.  Renaming or moving these files would be a breaking change for clients and that could be managed with semver, but it would be nice to be able to organize the package interior without causing that to be necessary.

Reply:
It should be a bit of a code smell, but since TestBox doesn't actually support modules yet, there's not really another way around it.  

> Renaming or moving these files

The globber usage inside Testbox is a black box implementation detail.  No end users should be knowing or caring about where those files are or that they even exist really.

Reply:
I meant clients of the globber module (other than TestBox).  I think I missing something about forgebox modules.  I'll look through the documentation today to try to catch up.